src/bridge/mod.rs

```diff
@@ -321,7 +317,7 @@ pub struct Bridge {
 
 pub fn start_bridge(
     ui_command_sender: LoggingTx<UiCommand>,
-    ui_command_receiver: RxUnbounded<UiCommand>,
+    ui_command_receiver: mpsc::UnboundedReceiver<UiCommand>,
```

Comment:
nit. Maybe just use mpsc::UnboundedReceiver rather than adding the namespace everywhere?